CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 Please see [CONTRIBUTING.md](https://github.com/cucumber/cucumber/blob/master/CONTRIBUTING.md) on how to contribute to Cucumber.
 
 ## [Unreleased]
+### Fixed
```

Comment:
It looks more like a breaking change than a fix

I see in option_splitter_spec that it seems absolute windows paths won't be working anymore if not prefixed with `file://`?

If I am mistaken, could it be possible to keep a test in option_splitter_spec that make sure absolute windows path are still working properly?

Reply:
> It looks more like a breaking change than a fix...

Yeah, sort of - the breaking change actually came in `8.0.0-rc.1` when we did the initial ESM support work and changed to using `import()` for loading custom classes. Unfortunately the option splitter didn't have awareness of `file://` so it was treating the colon as a delimiter, making it impossible to actually use a `file://` URL - once a valid `file://` URL makes it through to `FormatterBuilder`, it works as intended.

Happy to recast this as a breaking change though - or perhaps retroactively add a breaking change entry to the rc.1 changelog. What do you think?

> I see in option_splitter_spec that it seems absolute windows paths won't be working anymore if not prefixed with file://?

Yes, and same for unix absolute paths. We could try to preserve backwards compatibility with some pattern matching and then prepend `file://` internally, but honestly it feels cleaner and clearer to just say "absolute paths must be a valid file URL". It's definitely an edge case - I'd expect the majority of formatters to be within a file or package local to the project.

Reply:
> > It looks more like a breaking change than a fix...
> 
> Yeah, sort of - the breaking change actually came in `8.0.0-rc.1` when we did the initial ESM support work and changed to using `import()` for loading custom classes. Unfortunately the option splitter didn't have awareness of `file://` so it was treating the colon as a delimiter, making it impossible to actually use a `file://` URL - once a valid `file://` URL makes it through to `FormatterBuilder`, it works as intended.
> 
> Happy to recast this as a breaking change though - or perhaps retroactively add a breaking change entry to the rc.1 changelog. What do you think?
> 

Yeah, maybe adding it retroactively to rc.1 would do the trick :+1: 

> > I see in option_splitter_spec that it seems absolute windows paths won't be working anymore if not prefixed with file://?
> 
> Yes, and same for unix absolute paths. Again this was the case since `8.0.0-rc.1`. We could try to preserve backwards compatibility with some pattern matching and then prepend `file://` internally, but honestly it feels cleaner and clearer to just say "absolute paths must be a valid file URL". It's definitely an edge case - I'd expect the majority of formatters to be within a file or package local to the project.

I am okay with using "file://" only. It was just to be sure to have a full understanding of the changes

Reply:
Thanks, done that changelog update.